knotx-core/src/main/java/io/knotx/dataobjects/FileData.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (C) 2017 Cognifide Limited
```

Comment:
Actually, if I understand the idea correctly - this is an `inception year`. And 2016 as Knot.x inception year is ok.
In [this post](http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/) you can read a wider discussion about the topic.